cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -898,7 +898,7 @@ def _validate_sparsity(self):
             matrix_format = get_matrix_format(self.adata, x)
             if matrix_format == "csr":
                 continue
-            assert format != "unknown"
+            assert matrix_format != "unknown"
```

Comment:
@bkmartinjr tagging you as the (un)lucky introducer of this line only because you might have context for ramifications that are not immediately apparent to me, i.e., in case we were ever actually relying on that assert to fail. 🙏